### ISSUE TITLE
Export only doing html

### ISSUE THREAD

**Issue by sjDCC** [DMPRoadmap/roadmap#197]
When you click to export from either the 'my plans' page table, or the 'export' tab when editing a plan, both just display the html output (see screengrab 1). The export options are all missing (screengrab 2)

![capture](https://cloud.githubusercontent.com/assets/6169940/24197588/1ec9bf20-0efa-11e7-8350-cd732ccbf5fc.PNG)

![capture 2](https://cloud.githubusercontent.com/assets/6169940/24197591/22620b7e-0efa-11e7-9576-5dd12649130a.png)

**Comment by sjDCC** [comment]
Update: clicking 'export' from the plan details page does bring up a pop-up with the formatting options and dropdown to select export type
![export](https://cloud.githubusercontent.com/assets/6169940/24204966/faea2d98-0f11-11e7-990a-afea22728a58.PNG)

**Comment by jollopre** [comment]
The export functionality contains different formatting options (e.g. font size, type, margins, which sections to include, etc...) and a dropdown list to select the export format (e.g. CSV, HTML, JSON, pdf, text, xml and docx). The functionality should be implemented as:

- Tab page which should be displayed when showing a specific plan (e.g. plans#show). This tab is reached after clicking Export tab at plans#show view or after clicking Export link at (e.g. plans#index) view for a specific plan. 

- Popup which is triggered after clicking Export button at plans#show view (if user has edit permissions). This button is adjacent to the 'Answer questions' button.

**Comment by jollopre** [comment]
- The workflow for exporting has been fixed with commit (c3678ad).

- The exporting options functionality (e.g. choosing admin details fields, sections or formatting for pdf) was displaying every section for plan rather than those specific for a phase. Additionally, the admin details fields selected was not working either. Commit (7a0fc55) fixes it. However, clicking saving/reset through popup redirect the user to show_export page. **This will need revisit to be improved.**

- The only format that is currently working is HTML, fixed at commit (9a750e9). The rest of the formats still reference to the old model project.

Should I create an issue to solve the other formats? or do you want me to fix them through this issue?

**Comment by vyruss** [comment]
@jollopre I'd rather we put it in another issue and pass this to UAT

**Comment by sjDCC** [comment]
Thanks @jollopre @vyruss I'm not sure it will need UAT input as we're not changing anything, just re-implementing what we already have. Probably sensible to separate each export format into a different ticket though if none are working

**Comment by jollopre** [comment]
I have created the issue #222 in order to address the other formats. If it requires to do a lot of work, we could split into the different formats later on.

**Comment by vyruss** [comment]
Blocked by #222

**Comment by stephaniesimms** [comment]
Working as expected, except pdf export throws an error (see related issue #222 )
